i18n/en/cosmic_settings.ftl

```diff
@@ -233,6 +224,20 @@ x-hours = { $number ->
     *[other] { $number } hours
 }
 
+## Desktop: Window management
+
+window-management = Window management
```

Comment:
There is already a key for window-management on line 69, so you can add the new description there.

Reply:
Yeah I just noticed. I added -appearance to the one in Appearance. Let me know if it should be done differently.

Reply:
Also should hot-corner be removed, or is it planned to be added at some point?

Reply:
I think it is ok to leave it in for now